app/app.iml

```diff
@@ -95,5 +95,11 @@
     </content>
     <orderEntry type="jdk" jdkName="Android API 15 Platform" jdkType="Android SDK" />
     <orderEntry type="sourceFolder" forTests="false" />
```

Comment:
That's right. Android projects often don't need quite the same level of testing as Rails projects (for example), largely because apps don't tend to have a large surface area to manually test, and because compilation catches straight-up syntactical errors.